float64 x_cm
float64 y_cm
float64 z_cm
float64 size_cm
